pkgs/desktops/cinnamon/cinnamon-menus/default.nix

```
@@ -0,0 +1,24 @@
+{ fetchFromGitHub, glib, gobjectIntrospection, meson, ninja, pkgconfig, stdenv, wrapGAppsHook, libxml2, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "cinnamon-menus";
+  version = "4.4.0";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "0q4qj28swi2y93fj7pfil68l2cf9gmhbk6jmr8d70l54xf7sigsh";
+  };
+
+  buildInputs = [ glib gobjectIntrospection pkgconfig ];
+  nativeBuildInputs = [ meson ninja wrapGAppsHook cmake ];
+
+  meta = {
+    homepage = "http://cinnamon.linuxmint.com";
+    description = "The cinnamon-menu library";
```

This description isn't great
```suggestion
    description = "A menu system for the Cinnamon project";
```

it's also nixpkgs convention to not include the software's name in the description.